tutorials/hello/hello_client/src/app/plugins/hello_info/hello_info_plugin.ts

```diff
@@ -0,0 +1,111 @@
+/* eslint-disable @typescript-eslint/unbound-method */
+/* eslint-disable no-console */
+/* eslint-disable prefer-destructuring */
+/* eslint-disable dot-notation */
+/* eslint-disable @typescript-eslint/no-unsafe-call */
+/* eslint-disable @typescript-eslint/no-floating-promises */
+/* eslint-disable @typescript-eslint/no-unsafe-member-access */
+/* eslint-disable @typescript-eslint/no-unsafe-return */
+/* eslint-disable @typescript-eslint/no-unsafe-assignment */
+
+import { BasePlugin, db as liskDB, PluginInitContext, codec } from 'lisk-sdk';
+import {
+	getDBInstance,
+	getLastCounter,
+	getLastEventHeight,
+	setEventHelloInfo,
+	setLastCounter,
+	setLastEventHeight
+} from './db';
+import { configSchema, chainEventSchema } from './schemas';
+import { HelloInfoPluginConfig } from './types';
+import { Endpoint } from './endpoint';
+
+export class HelloInfoPlugin extends BasePlugin<HelloInfoPluginConfig> {
+	public configSchema = configSchema;
+	public endpoint = new Endpoint();
+	public counter = 0;
+	private height = 0;
+	private _pluginDB!: liskDB.Database;
+
+	public get nodeModulePath(): string {
+		return __filename;
+	}
+
+	public async init(context: PluginInitContext): Promise<void> {
+		await super.init(context);
+	}
+
+	// loads DB instances and initiates counter for first run.
+	public async load(): Promise<void> {
+		this._pluginDB = await getDBInstance(this.dataPath);
+		this.endpoint.init(this._pluginDB);
+		const lastCounter = await getLastCounter(this._pluginDB);
+		if (lastCounter.counter > 0) {
+			// Syncs plugin's database after an interval.
+			setInterval(() => { this._syncChainEvents(); }, this.config.syncInterval);
+		} else {
+			const newCounter = 0;
+			await setLastCounter(this._pluginDB, newCounter);
+			// Syncs plugin's database after an interval.
+			setInterval(() => { this._syncChainEvents(); }, this.config.syncInterval);
+		}
+	}
```

Comment:
```suggestion
		} else {
			const newCounter = 0;
			await setLastCounter(this._pluginDB, newCounter);
		}
		// Syncs plugin's database after an interval.
		setInterval(() => { this._syncChainEvents(); }, this.config.syncInterval);
	}
```

move line below if-else, to remove duplicate lines